### PR TITLE
[NAS] fix bug on pdarts

### PR DESCRIPTION
**Pull request by squirrelsc**
After last refactoring, the network cannot be reduced correctly before training. With this fix, it can remove low performance ops correctly.

### CODE REVIEW

**Comment by ultmaster** [comment]
Please merge into v1.2

File: src/sdk/pynni/nni/nas/pytorch/pdarts/mutator.py [hunk 1]
```diff
@@ -25,13 +27,13 @@ def __init__(self, model, pdarts_epoch_index, pdarts_num_to_drop, switches={}):
         for mutable in self.mutables:
             if isinstance(mutable, LayerChoice):
 
-                switches = self.switches.get(mutable.key, [True for j in range(mutable.length)])
+                switches = self.switches.get(mutable.key, [True for j in range(mutable.length+1)])
+                switches.requires_grad = False
+                choices = self.choices[mutable.key]
 
-                for index in range(len(switches)-1, -1, -1):
+                for index in range(len(switches)-2, -1, -1):
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/nas/pytorch/pdarts/mutator.py]: better to add comment to explain why `-2`

Comment (squirrelsc) [reply src/sdk/pynni/nni/nas/pytorch/pdarts/mutator.py]: added

Comment (squirrelsc) [reply src/sdk/pynni/nni/nas/pytorch/pdarts/mutator.py]: new logic is simpler, so no -2 is here.

**Pull request merged**